### ISSUE TITLE
Possible NPE in VertxImpl::deployVerticle

### ISSUE THREAD

**Issue by juanavelez** [eclipse/vert.x#1383]
```
  @Override
  public void deployVerticle(Verticle verticle, DeploymentOptions options, Handler<AsyncResult<String>> completionHandler) {
    boolean closed;
    synchronized (this) {
      closed = this.closed;
    }
    if (closed) {
      completionHandler.handle(Future.failedFuture("Vert.x closed"));
    } else {
      deploymentManager.deployVerticle(verticle, options, completionHandler);
    }
  }
```

This method is called with null argument for `completionHandler` by at least 2 other VertxImpl methods. However, there is no check for null when calling completionHandler when Vertx is closed.

**Comment by vietj** [comment]
would worth a check and skip the completionHandler callback

**Comment by vietj** [comment]
fancy a contribution ?

**Comment by juanavelez** [comment]
I’d love to but my (US) employer won’t allow me to sign off on the contributor agreement. Why? They did not say.

Sorry,

Thanks - Juan

**Comment by cescoffier** [comment]
This issue has been fixed by #1422.
